Article/Article.js

```diff
@@ -26,4 +26,5 @@ class Article {
 
 */
 
-let articles;
\ No newline at end of file
+let articles = document.querySelectorAll('.article');
+articles.forEach(eachArticle => new Article(eachArticle));
```

Comment:
Very slick, I like this especially using the nuances of the Fat Arrows implicit return and single parameter syntax.